app/controllers/code_ocean/files_controller.rb

```diff
@@ -66,7 +66,7 @@ def create_and_respond(options = {})
         else
           filename = "#{@object.path || ''}/#{@object.name || ''}#{@object.file_type.try(:file_extension) || ''}"
           format.html do
-            flash[:danger] = t('files.error.filename', name: filename)
+            flash[:danger] = t('code_ocean/files.error.filename', name: filename)
```

Comment:
The prefix is interesting. I'm not really used to this, using `/`.

Reply:
Yes, I seldomly use it, too. However, since the class is called `CodeOcean::File` (and not just `File` due to the conflict with the internal class representing access to objects on the filesystem), we need to use it everywhere... This is actually the prefix Rails expect if using the `CodeOcean::File.model_name.human` method. 🤷